public/js/bootstrap-dropdown.js

```diff
@@ -66,7 +66,7 @@
    * =================================== */
 
   $(function () {
-    $('html').on('click.dropdown.data-api', clearMenus!function( $ ){
+    $('html').on('click.dropdown.data-api', clearMenus, function( $ ){
```

Comment:
is this file part of Bootstrap?

Reply:
I'm not 100% sure, it's been in the repo for a while and was just bringing up a syntax error in the console, so I fixed it.

Reply:
ok, the `!function` syntax basically declares and then runs a function iirc but since it's a callback it should still execute normally

Reply:
sounds good